### DISCUSSION TITLE
could add more type edit box

### DISCUSSION THREAD

**Discussion by bruce2014** [juliusgb/obsidian-tldraw-plugin#63] [Ideas]
could add more type edit box ， Hexagon、heart-shaped、circled、five-pointed star

just like the website did：
https://www.tldraw.com/

**Comment by juliusgb** [comment]
Thanks for the idea.

tldraw.com uses tldraw version 2 (has more features and shapes) while this plugin still uses tldraw version 1 (https://github.com/tldraw/tldraw-v1).
